src/test/java/net/sf/jabref/importer/fileformat/CopacImporterTest.java

```diff
@@ -75,17 +95,56 @@ public void testImportEntries() throws IOException {
-                .getResourceAsStream("CopacImporterTest2.txt")) {
+        try (InputStream stream = CopacImporterTest.class.getResourceAsStream("CopacImporterTest2.txt")) {
             List<BibEntry> entries = importer.importEntries(stream, new OutputPrinterToNull());
             Assert.assertEquals(2, entries.size());
             BibEntry one = entries.get(0);
 
             Assert.assertEquals("Computing and operational research at the London Hospital", one.getField("title"));
+            Assert.assertEquals("Barber, Barry and Abbott, W.", one.getField("author"));
+            Assert.assertEquals("Computers in medicine series", one.getField("series"));
+            Assert.assertEquals("London :  Butterworths", one.getField("publisher"));
+            Assert.assertEquals("x, 102p, leaf : ill., form, port ; 22cm", one.getField("physicaldimensions"));
+            Assert.assertEquals("0407517006 (Pbk)", one.getField("isbn"));
+            Assert.assertEquals("Bibl.p.94-97. - Index", one.getField("note"));
+            Assert.assertEquals(
+                    "London Hospital and Medical College, Electronic data processing - Medicine, Computers - Hospital administration, Hospital planning, Operations research, Hospital equipment and supplies, Electronic data processing - Hospitals - Administration, Hospitals, London, London Hospital and Medical College, Records management, Applications of computer systems, to 1971",
+                    one.getField("keywords"));
+            Assert.assertEquals("TXT, PDF", one.getField("documenttype"));
+            Assert.assertEquals(
+                    "Aberdeen ; Birmingham ; Edinburgh ; Trinity College Dublin ; UCL (University College London)",
+                    one.getField("HL"));
+
+            Assert.assertEquals("book", one.getType());
 
             BibEntry two = entries.get(1);
 
             Assert.assertEquals("Real time systems : management and design", two.getField("title"));
+            Assert.assertEquals("Tebbs, David and Collins, Garfield", two.getField("author"));
+            Assert.assertEquals("London ; New York :  McGraw-Hill", two.getField("publisher"));
+            Assert.assertEquals("1977", two.getField("year"));
+            Assert.assertEquals("ix, 357p : ill., forms ; 24cm", two.getField("physicaldimensions"));
+            Assert.assertEquals("0070844828", two.getField("isbn"));
+            Assert.assertEquals("Index", two.getField("note"));
+            Assert.assertEquals("Real-time data processing - Management, Real time computer systems, Design",
+                    two.getField("keywords"));
+            Assert.assertEquals("TXT, PDF", two.getField("documenttype"));
+            Assert.assertEquals(
+                    "Aberdeen ; Birmingham ; Edinburgh ; Imperial College ; Liverpool ; Manchester ; Oxford ; Trinity College Dublin",
+                    two.getField("HL"));
+
+            Assert.assertEquals("book", two.getType());
```

Comment:
Please have a look at how the other successfully merged PRs compared bibentries after the import...